src/common/AndroidUtils.ts

```diff
@@ -154,157 +139,147 @@ export class AndroidSDKUtils {
         }
 
         if (!AndroidSDKUtils.isCached()) {
```

Comment:
Minor: with the switch to using more of an async pattern here, it might read better/less "indented" to switch to a quick return pattern instead:
```typescript
if (AndroidSDKUtils.isCached()) {
    return Promise.resolve(AndroidSDKUtils.packageCache);
}

return CommonUtils.executeCommandAsync(
// ...
```